beep/features/featurizer_helpers.py

```diff
@@ -91,7 +91,7 @@ def isolate_dQdV_peaks(processed_cycler_run, diag_nr, charge_y_n, max_nr_peaks,
 
 def generate_model(spec):
     """
-    Method that generates a model to fit the hppc data to for peak extraction, using spec dictionary
+    Method that generates a model to fit the RPT data to for peak extraction, using spec dictionary
```

Comment:
Update docstring to fix typo: "[...] fit the RPT data for peak extraction [...]"